src/game.c

```diff
@@ -774,9 +778,17 @@ int game_apply_theme(void)
 		lay = txt_layout(game_theme.font, align, game_theme.win_w, game_theme.win_h);
 		if (!lay)
 			return -1;
+
 		box = txt_box(game_theme.win_w, game_theme.win_h);
 		if (!box)
 			return -1;
+
+		#ifdef _USE_HARFBUZZ
+		/* Use the game language direction as inital value. */
+		if (g)
+			txt_layout_direction(lay, g->rtl? HB_DIRECTION_RTL: HB_DIRECTION_LTR);
```

Comment:
sry for my pedantic review, but why not just pass g->rtl to txt_laout_direction?
Like this:
txt_layout_direction(lay, g ? g->rtl : 0);
Pros: 
1) no if
2) no HARFBUZZ #ifdef needed
3) game.c code is not deal with specific library
4) no = 4 hardcoded constant.

And then, in graphics.c you can convert g->rtl to HB_ *macroses.

Reply:
The main idea:
- graphics.c can include library specific code
- game.c should avoid library specific code

Reply:
It sounds good to me! I'll apply your suggestion.

Reply:
btw, `g->rtl` is only a boolean flag. However `layout-direction`, if compiled with HarfBuzz, is of type `hb_direction_t`: 
```
typedef enum {
  HB_DIRECTION_INVALID = 0,
  HB_DIRECTION_LTR = 4,
  HB_DIRECTION_RTL,
  HB_DIRECTION_TTB,
  HB_DIRECTION_BTT
} hb_direction_t;```

Reply:
Yes. And i mean that it can be boolean in layout, line, word too and converted to enum while rendering. If there is only two flags used: rtl/ltr. Line, world and layout are abstract structures. At least we must try to make them abstract.

Reply:
I applied those changes, please see whether it is okay.

Reply:
> If there is only two flags used: rtl/ltr

It is possible to make it a flag but I think direction is more than a flag. HarfBuzz has an enum with five entries for it. Including top to botom and botom to top and invalid.